packages/native/types/tsconfig.json

```diff
@@ -6,14 +6,11 @@
     "lib": ["es6", "dom"],
     "module": "commonjs",
     "noEmit": true,
-    "noImplicitAny": true,
-    "noImplicitThis": true,
     "strict": true,
-    "strictNullChecks": true,
-    "strictFunctionTypes": true,
     "target": "es5",
     "typeRoots": ["../"],
-    "types": []
+    "types": [],
+    "skipLibCheck": true
```

Comment:
I'm happy with this as this kinda should IMHO be default anyway but I wonder why this has been added now? Were new errors raised now somehow after the dtslint upgrade?

Reply:
Was getting some weird error from internal React Native types. Was going to get the exact error message but saw you already merged 😉

Had to make the same change to `@emotion/server` because `@emotion/server` does not have DOM types available, but it imports stuff that uses DOM types.